client/src/components/import-component/import-component.jsx

```diff
@@ -0,0 +1,203 @@
+import { useState, useEffect, useRef } from "react";
+import {
+  Typography,
+  Stack,
+  Card,
+  CardHeader,
+  CardContent,
+  Select,
+  MenuItem,
+  FormHelperText,
+  Button,
+  FormControl,
+  InputLabel,
+  Paper,
+  Grid,
+  IconButton,
+  LinearProgress,
+  Box,
+} from "@mui/material";
+import CloseIcon from "@mui/icons-material/Close";
+import { Form, Header, Upload, UploadIcon, Cancel } from "./index.styles";
+
+function importComponent() {
+  const [menu, setMenu] = useState("");
+  const [files, setFiles] = useState([]);
+  const [progress, setProgress] = useState(0);
+
+  useEffect(() => {
+    const timer = setInterval(() => {
+      setProgress((oldProgress) => {
+        // if (oldProgress === 100) {
+        //   return 0;
+        // }
+        const diff = Math.random() * 10;
+        return Math.min(oldProgress + diff, 100);
+      });
+    }, 500);
+
+    return () => {
+      clearInterval(timer);
+    };
+  }, []);
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    // Grab the file
+    const { files: currFiles } = e.target.files;
+    // Create a new FormData object
+    const formData = new FormData();
```

Comment:
Does this handle reading the file and converting it to an acceptable JSON that can be passed to the backend? The bulk addition of object will require the frontend to handle the conversion of the CSV to an acceptable list of JSON objects.

Reply:
We have a meeting with Wilfred tomorrow and we will be able to get some specific information on that data that can be expected. We will provide some of the data examples